Integrations/integration-ServiceNowPython.yml

```diff
@@ -0,0 +1,1728 @@
+            'Contents': res,
+            'ContentsFormat': formats['json'],
+            'ReadableContentsFormat': formats['markdown'],
+            'HumanReadable': tableToMarkdown('File Uploaded', hr),
+            'EntryContext': {
+                'Ticket(val.ID==obj.ID)' : context
+            }
+        }
+
+        return entry
+
+
+    def upload_file(ticket_id, file_id, file_name, ticket_type):
+        headers = {
+            'Accept': 'application/json'
+        }
+
+        body = {
+            'table_name': ticket_type,
+            'table_sys_id': ticket_id,
+            'file_name': file_name
+        }
+
+        path = 'attachment/upload'
+
+        return send_request(path, 'post', headers = headers, body = body, file = {'id': file_id, 'name': file_name})
+
+
+    def get_computer_command():
+        ticket_type = 'cmdb_ci_computer'
+        computer_name = demisto.args()['computerName']
+
+        res = get_computer(ticket_type, computer_name)
+
+        if not res or 'result' not in res:
+            return 'Cannot find computer'
+        elif isinstance(res['result'], list):
+            if len(res['result']) == 0:
+                return 'Cannot find computer'
+            computer = res['result'][0]
+        else:
+            computer = res['result']
+
+        if computer['u_code'] != computer_name:
+            return 'Computer not found'
+
+        hr = {
+            'Id': computer['sys_id'],
+            'u_code (computer name)': computer['u_code'],
+            'Support group': computer['support_group'],
+            'Operating System': computer['os'],
+            'Comments': computer['comments']
+        }
+
+        ec = createContext(computer, removeNull = True)
+        ec['support_group'] = computer['support_group']['value'] if 'support_group' in computer else ''
+
+        entry = {
+            'Type': entryTypes['note'],
+            'Contents': computer,
+            'ContentsFormat': formats['json'],
+            'ReadableContentsFormat': formats['markdown'],
+            'HumanReadable': tableToMarkdown('ServiceNow Computer', hr),
+            'EntryContext': {
+                  'ServiceNowComputer(val.sys_id==obj.sys_id)': ec,
+            }
+        }
+
+        return entry
+
+
+    def get_computer(ticket_type, computer_name):
+        path = 'table/' + ticket_type
+
+        query_params = {
+            'sysparm_query': 'u_code=' + computer_name
+        }
+
+        return send_request(path, 'get', params=query_params)
+
+    def get_groups_command():
+        ticket_type = 'sys_user_group'
+        group_name = demisto.args()['name']
+        res = get_groups(ticket_type, group_name)
+
+        if not res or 'result' not in res:
+            return 'Cannot find groups'
+
+        hr_groups = []
+        context_groups = []
+
+        for group in res['result']:
+            if group['name'] == group_name:
+                hr_groups.append({
+                    'Id': group['sys_id'],
```

Comment:
should be ID

Reply:
backward compatibility, can add ID in addition